plugin/skills/create-ui-design/SKILL.md

```
@@ -0,0 +1,264 @@
+1. Issue に理由（実行したコマンドとエラー出力の要約）をコメントする
+   ```bash
+   gh issue comment $0 --body-file - <<'EOF'
+   ## Pencil を利用できないためデザインを作成できません（要人手確認）
+
+   `create-ui-design` は Pencil CLI が必要ですが、実行環境で利用できませんでした。
+
+   ## 実行したコマンドとエラー
+
+   ```
+   <コマンドとエラー出力の要約>
+   ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)